osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -241,12 +240,10 @@ protected override bool CheckLocalAvailability(BeatmapSetInfo model, IQueryable<
         /// <param name="info">The <see cref="BeatmapInfo"/> to save the content against. The file referenced by <see cref="BeatmapInfo.Path"/> will be replaced.</param>
         /// <param name="beatmapContent">The <see cref="IBeatmap"/> content to write.</param>
         /// <param name="beatmapSkin">The beatmap <see cref="ISkin"/> content to write, null if to be omitted.</param>
-        public void Save(BeatmapInfo info, IBeatmap beatmapContent, ISkin beatmapSkin = null)
+        public virtual void Save(BeatmapInfo info, IBeatmap beatmapContent, ISkin beatmapSkin = null)
         {
             var setInfo = info.BeatmapSet;
 
-            Debug.Assert(setInfo.Files != null);
```

Comment:
Are you sure about this removal? What's the case that this comes up?

Reply:
Nevermind, was removed because it's "non-nullable". Though the property can be re-assigned...